_charas/giansar.md

```diff
@@ -42,8 +42,7 @@ salesEnd = "Exactly on time, that felt good"
 battleStart = "These guys are far from perfect. I'll keep them all under my control."
 action = "Who's next?"
 attack = "Hmph!"
-skill1 = "It's full of gaps."
-skill2 = "I'll rewrite it"
+skill = "It's full of gaps." 
```

Comment:
You can use the field names skillA, skillB if the character has two skill voice lines